youth-leading-the-way.html

```diff
@@ -0,0 +1,536 @@
+              <a
+                href="#"
+                class="collapsible menu__link"
+                aria-haspopup="true"
+                aria-expanded="false"
+                role="button"
+                >Resources &#9660;</a
+              >
+              <ul class="nav subnav submenu">
+                <li class="subnav-link">
+                  <a class="nav-link" href="volunteer-opportunities.html"
+                    >Volunteer Opportunities</a
+                  >
+                </li>
+                <li class="subnav-link">
+                  <a class="nav-link" href="covid19.html">Covid Response</a>
+                </li>
+                <li class="subnav-link">
+                  <a
+                    class="nav-link"
+                    class="page-scroll"
+                    href="covid19.html#anti-black_racism_guidebook"
+                    >Anti-Black Racism Guidebook</a
+                  >
+                </li>
+              </ul>
+            </li>
+            <li class="subnav-menu">
+              <a
+                href="#"
+                class="collapsible menu__link"
+                aria-haspopup="true"
+                role="button"
+                aria-expanded="false"
+                >Get Involved &#9660;</a
+              >
+              <ul class="nav subnav">
+                <li class="subnav-link">
+                  <a class="nav-link" href="jointeam.html">Join the Team</a>
+                </li>
+                <li class="subnav-link">
+                  <a class="nav-link" href="donate.html">Donate</a>
+                </li>
+                <li class="subnav-link">
+                  <a class="nav-link" href="lendahandsurvey.html">Lend a Hand Survey</a>
+                </li>
+                <li class="subnav-link">
+                <a class="nav-link" href="youth-leading-the-way.html">Find your volunteer opportunity</a>
+              </li>
+              </ul>
+            </li>
+            <li>
+              <a class="nav-link" href="workshops.html">Workshops</a>
+            </li>
+            <li>
+              <a href="./blog">Blog</a>
+            </li>
+            <li>
+              <a class="page-scroll" href="contactus.html">Contact Us</a>
+            </li>
+            <li class="nav-button">
+              <a class="btn btn-seconday btn-sm" href="index.html#help">
+                How can we help?
+              </a>
+            </li>
+            <!-- <a href="index.html#help" class="btn btn-seconday btn-sm" style="margin-left: 12px; margin-top: 12px;">How can we help?</a> -->
+          </ul>
+        </div>
+        <!-- /.navbar-collapse -->
+      </div>
+      <!-- /.container-fluid -->
+    </nav>
+  
+      </div>
+
+      <!-- header section -->
+      <div class="first-div">
+        <div class="first-landing-div">
+          <h2 class="help-title main-title">YOUTH LEADING THE WAY</h2>
+          <button class="header-button">PROGRAM</button>
```

Comment:
Are you sure this button is supposed to do nothing at all?  If you want the aesthetic of a button, but it doesn't do anything when clicked, use div.